### PR TITLE
Make meta serializable

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1129 https://github.com/webstudio-is/webstudio-builder/issues/1545

Here put all components meta icons into strings and fixed usage with innerHTML. This allows us to load components only from canvas and send to builder.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by istarkov — APPROVED** [review]

File: apps/builder/app/builder/features/props-panel/props-panel.tsx [hunk 1]
```diff
@@ -59,7 +59,10 @@ const InstanceInfo = ({
     css={{ height: theme.spacing[13], color: theme.colors.foregroundSubtle }}
     align="center"
   >
-    <meta.Icon />{" "}
+    <div
+      style={{ width: 16, height: 16 }}
+      dangerouslySetInnerHTML={{ __html: meta.Icon }}
```
Comment (kof) [inline apps/builder/app/builder/features/props-panel/props-panel.tsx:64]: if icon is a html string, then it shouldn't be upper case any more?

File: apps/builder/app/builder/features/sidebar-left/panels/components/components.tsx [hunk 2]
```diff
@@ -110,7 +110,11 @@ export const TabContent = ({ publish, onSetActiveTab }: TabContentProps) => {
                             {...pressProps}
                             {...{ [dragItemAttribute]: component }}
                             label={meta.label}
-                            icon={<meta.Icon />}
+                            icon={
+                              <div
```
Comment (kof) [inline apps/builder/app/builder/features/sidebar-left/panels/components/components.tsx:114]: maybe we could just move the rendering into the final component and pass the icon as-is as a string?

**Review by kof — APPROVED** [review]

File: apps/builder/app/builder/shared/meta-icon.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+type Props = {
+  size?: number | string;
+  icon: string;
+};
+
+export const MetaIcon = ({ size = 16, icon }: Props) => {
+  return (
+    <div
+      style={{ width: size, height: size }}
```
Comment (kof) [inline apps/builder/app/builder/shared/meta-icon.tsx:9]: I would not use inline style, since you can have reusable css

Comment (kof) [reply apps/builder/app/builder/shared/meta-icon.tsx:9]: Would probably use a span, but no strong opiinion

Comment (kof) [reply apps/builder/app/builder/shared/meta-icon.tsx:9]: check out https://www.radix-ui.com/docs/primitives/utilities/accessible-icon

Comment (kof) [reply apps/builder/app/builder/shared/meta-icon.tsx:9]: depending on how icon is used, you may need to announce it accessibly and/or hide it from screen readers check out what radix does with icons

**Pull request merged**